.github/ISSUE_TEMPLATE.md

```diff
@@ -31,11 +31,11 @@ Note that if you do not provide enough information to reproduce the issue, we ma
 <!--
 Providing an isolated reproduction of the bug in a codepen makes it much easier for us to help you. Here are some ways to get started:
 
-  * Go to https://aka.ms/fabricpen for a starter codepen
+  * Go to https://aka.ms/fluentuipen for a starter codepen
```

Comment:
Could we do `fluentpen`? It's easier to type. 😆

Reply:
yeah...sure :D i agree on the easier part for sure.